### PR TITLE
Add “mail³” to the dapps list

### PR DESCRIPTION
**Pull request by qiudeqiu**
Hope to join your dapps list, thanks

### CODE REVIEW

File: app/src/main/assets/dapps_list.json [hunk 1]
```diff
@@ -87,4 +87,5 @@
   {"name": "MetaPath", "description": "Assets can be routed across different blockchains", "url": "https://dapp.path.finance?source=alpha", "category": "Exchange"},
   {"name": "SWFT", "description": "One Stop Cross-Chain swap, supporting 300+ tokens on 30+ chains", "url": "https://defi.swft.pro?sourceFlag=alpha", "category": "Exchange"},
   {"name": "Swapzone", "description": "Swapzone is a custody-free cryptocurrency exchange aggregator with no registration needed.", "url": "https://swapzone.io/", "category": "Exchange"}
+  {"name": "mail³", "description": "Decentralized mail system for web3 natives.", "url": "https://app.mail3.me/", "category": "Social Media"}
```
Comment (JamesSmartCell) [inline app/src/main/assets/dapps_list.json:90]: Need to add a comma after the previous line like this:

```
{"name": "Swapzone", "description": "Swapzone is a custody-free cryptocurrency exchange aggregator with no registration needed.", "url": "https://swapzone.io/", "category": "Exchange"}, //<--- comma
{"name": "mail³", "description": "Decentralized mail system for web3 natives.", "url": "https://app.mail3.me/", "category": "Social Media"}
```

**Review by JamesSmartCell — CHANGES_REQUESTED** [review]
@qiudeqiu you need to add a comma as directed for this to build.

File: app/src/main/assets/dapps_list.json (hunk 1 above)
Comment (qiudeqiu) [reply app/src/main/assets/dapps_list.json:90]: Already modified，thanks for reviewing

**Review by JamesSmartCell — APPROVED** [review]

**Comment by qiudeqiu** [comment]
Hi, we're glad to have been added to the alphawallet_dapp_list through the review ~ we have a question, we don't seem to see the dapp display on the Android alphawallet browser, is there any process we're missing? thanks

**Comment by seabornlee** [comment]
> Hi, we're glad to have been added to the alphawallet_dapp_list through the review ~ we have a question, we don't seem to see the dapp display on the Android alphawallet browser, is there any process we're missing? thanks

Hi, no more process, it will take effect in the next release.

**Pull request merged**